test/autotest.py

```
@@ -1572,7 +1768,7 @@ def _build_tests(self) -> List[TestSpec]:
                 "screen", 3,
                 ["/usr/bin/screen -c /dev/null -s /bin/sh"],
                 env={"TERM": "vt100"},
-                private_env_dirs={"SCREENDIR": "screen"},
+                private_env_dirs={"SCREENDIR": "/tmp/{workname}-screen"},
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/{workname}-screen"

(S108)